components/ProposalRow.tsx

```diff
@@ -34,6 +38,27 @@ export default function ProposalRow({
     args: [id],
     value: fees?.l1 || BigInt(0),
   });
+  const bridgeButton =
+    bridged !== null && !proposalMetadataBridgeIsLoading ? (
+      <div className="rounded bg-gray-500 px-2 py-1 text-xs font-semibold text-white shadow-sm focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600">
+        Waiting for relayer 🕓
+      </div>
+    ) : (
+      <button
+        className="rounded bg-indigo-600 px-2 py-1 text-xs font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"
+        onClick={async () => {
+          await walletClient!.switchChain({ id: l1.chain.id });
```

Comment:
When on optimism goerli, clicking `Bridge Proposal` -> `Switch Network` on Metamask popup -> causes the following error:
<img width="887" alt="image" src="https://github.com/ScopeLift/l2-flexible-voting-frontend/assets/61768337/6ff32d14-2f41-4cc9-99bd-7902f53d5622">